src/main/export.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-plusplus */
+import { join } from 'path';
+import { Project, Transcription } from '../sharedTypes';
+import { padZeros, integerDivide, writeFile, mkdir } from './util';
+
+const secondToTimestamp = (num: number) => {
+  return `${padZeros(integerDivide(num, 3600), 2)}:
+          ${padZeros(integerDivide(num, 60), 2)}:
+          ${padZeros(integerDivide(num, 1), 2)}:
+          ${padZeros(integerDivide(num, 0.01), 2)}`;
+};
+
+const constructEDL = (title: string, transcription: Transcription) => {
+  let output = `TITLE: ${title}\nFCM: NON-DROP FRAME\n\n`;
+  try {
+    const { words } = transcription;
+    const entries = words.length;
+    for (let i = 0; i < entries; i++) {
```

Comment:
Could be cleaner to do this with `map` since every iteration of the loop is just producing a new line of output

Reply:
Got it